helm/values.yaml

```diff
@@ -79,6 +81,9 @@ rawSpansGrouperConfig:
     sink:
       kafka:
         bootstrapServers: "bootstrap:9092"
+        lingerMs: 0
+        batchSize: 16384
+        compressionType: none
```

Comment:
we have overrides in prod for these am assuming ?

Reply:
yes. will raise a PR in helm-values repo.